### ISSUE TITLE
Blueutil crashing after running the workflow on MacOS Sonoma

### ISSUE THREAD

**Issue by darthlite** [Acidham/alfred-airpod-connector#20]
I have upgraded to Sonoma, and when I run the workflow, I get an error from MacOS saying blueutil crashed and asking me if I want to send a report. This seems to be the key error in the report:

"This app has crashed because it attempted to access privacy-sensitive data without a usage description. The app's Info.plist must contain an NSBluetoothAlwaysUsageDescription key with a string value explaining to the user how the app uses this data."

I'm not sure if this is a blueutil or workflow problem, but as far as I can tell, blueutil works perfectly from the command line and I don't get this error.

Any advice on what the cause might be (or how to debug/fix it) is appreciated.

**Comment by Acidham** [comment]
Strange. I am also on Sonoma, but I cannot reproduce the error. The workflow and Blueutil is working perfectly. When you can exec blueutil from command line, another application seems to cause the issue.

**Comment by darthlite** [comment]
Hmm, must be something on my end that is configured strangely and causing things to fail.

The only other thing I just noticed, and I'm not sure if this has anything to do with the crashing, is that the workflow is telling me my airpods are connected when they are not. I don't remember noticing it yesterday. Just mentioning in case it helps point me in the right direction.

I tried looking into Alfred's debugger, but that either outputs no errors/warnings or just says:

`STDERR: AirPods Connector[Run Script] <mac-address>`

**Comment by darthlite** [comment]
Okay, spent more time on this that I would like to admit, and finally found a fix. The problem turned out to be neither the workflow nor blueutil, but Alfred required Bluetooth permissions (wish it or MacOS told me that in some way!). As soon as I enabled that, it worked.

Not sure if enabling BT was a requirement, or if I did have it enabled but the update to Sonoma broke it, but in any case, all good now thankfully :)
